File: python/fundamentals/day-02/exercise2.py
def check_number(num):
   
    if num > 0:
        return "Positive"
    elif num < 0:
        return "Negative"
    else:
        return "Zero"

def main():
    try:
        # Read input from user
        user_input = input("Enter a number: ").strip()
        number = float(user_input)  # Accepts integers and decimals

        # Display result
        print(f"The number is {check_number(number)}.")

    except ValueError:
        print("Invalid input. Please enter a valid numeric value.")

if __name__ == "__main__":
    main()

File: java/fundamentals/day02/exercise2.java
package fundamentals.day02;


	import java.util.Scanner;

	public class exercise2 {
	    public static void main(String[] args) {
	        Scanner scanner = new Scanner(System.in);

	        System.out.print("Enter a number: ");
	        if (scanner.hasNextDouble()) { // Accepts integers and decimals
	            double number = scanner.nextDouble();

	            if (number > 0) {
	                System.out.println("The number is Positive.");
	            } else if (number < 0) {
	                System.out.println("The number is Negative.");
	            } else {
	                System.out.println("The number is Zero.");
	            }
	        } else {
	            System.out.println("Invalid input. Please enter a valid numeric value.");
	        }

	        scanner.close();
	    }
	}


